system/draft/test_misc.py

```diff
@@ -83,45 +84,76 @@
     await send_nym(pool_handle, wallet_handle, trustee_did, random_did)
+    await send_attrib(pool_handle, wallet_handle, trustee_did, random_did, None, json.dumps({'key': 'value'}), None)
     schema_id, _ = await send_schema(pool_handle, wallet_handle, trustee_did, random_string(10), '1.0',
                                      json.dumps([random_string(1), random_string(2), random_string(3)]))
     time.sleep(1)
     res = json.dumps(await get_schema(pool_handle, wallet_handle, trustee_did, schema_id))
     schema_id, schema_json = await ledger.parse_get_schema_response(res)
     cred_def_id, _, _ = await send_cred_def(pool_handle, wallet_handle, trustee_did, schema_json, random_string(3),
                                             None, json.dumps({'support_revocation': True}))
-    hosts = [testinfra.get_host('docker://node' + str(i)) for i in range(1, 5)]
+    revoc_reg_def_id, _, _, res1 = await send_revoc_reg_entry(pool_handle, wallet_handle, trustee_did, 'CL_ACCUM',
+                                                              random_string(3), cred_def_id,
+                                                              json.dumps({'max_cred_num': 1,
+                                                                          'issuance_type': 'ISSUANCE_BY_DEFAULT'}))
+    timestamp1 = int(time.time())
+
+    # uncomment to check freshness state proof reading
+    # time.sleep(600)
+
+    hosts = [testinfra.get_host('docker://node' + str(i)) for i in range(1, 8)]
     print(hosts)
     outputs0 = [host.run('systemctl stop indy-node') for host in hosts[:-1]]
     print(outputs0)
-
-    time.sleep(1)
     try:
         req1 = await ledger.build_get_nym_request(None, random_did)
         res1 = json.loads(await ledger.submit_request(pool_handle, req1))
 
-        req2 = await ledger.build_get_schema_request(None, schema_id)
+        req2 = await ledger.build_get_attrib_request(None, random_did, 'key', None, None)
         res2 = json.loads(await ledger.submit_request(pool_handle, req2))
 
-        req3 = await ledger.build_get_cred_def_request(None, cred_def_id)
+        req3 = await ledger.build_get_schema_request(None, schema_id)
         res3 = json.loads(await ledger.submit_request(pool_handle, req3))
+
+        req4 = await ledger.build_get_cred_def_request(None, cred_def_id)
+        res4 = json.loads(await ledger.submit_request(pool_handle, req4))
+
+        req5 = await ledger.build_get_revoc_reg_def_request(None, revoc_reg_def_id)
+        res5 = json.loads(await ledger.submit_request(pool_handle, req5))
+
+        # consensus is impossible with timestamp0 here! IS-1263
+        req6 = await ledger.build_get_revoc_reg_request(None, revoc_reg_def_id, timestamp1)
+        res6 = json.loads(await ledger.submit_request(pool_handle, req6))
+
+        # consensus is impossible with (timestamp0, timestamp1) here! IS-1264
+        req7 = await ledger.build_get_revoc_reg_delta_request(None, revoc_reg_def_id, None, timestamp1)
```

Comment:
We should add state proof tests for
- GET_AUTH_RULE (with parameters, and when the corresponding AUTH_RULE was sent to the ledger)
- GET_FEES
- GET_TAA
- GET_TAA_AML